### PR TITLE
Some code tweaks for brevity; fixed example env spelling

### PR DESCRIPTION
**Pull request by justetz**
* Reduced code copying with new helpers;
* Renamed `SPEADSHEET_ID` to `SPREADSHEET_ID` in `.env.example`;
* Created a `GDRIVE_EMAIL` env variable to enable use of different emails as needed; and
* General code adjustments for brevity.

### CODE REVIEW

**Comment by justetz** [comment]
Bump @ddbruce

File: .env.example [hunk 1]
```diff
@@ -1,7 +1,10 @@
-SPEADSHEET_ID=idhere
+SPREADSHEET_ID=idhere
 NODE_PORT=3000
 SLACK_BOT_TOKEN=xoxb-YOURBOTTOKENHERE
 SLACK_SIGNING_SECRET=YOURSECRETHERE
 CHORES_CHANNEL=channel-here
 CRON_SCHEDULE=* * * * *
 TZ="America/New_York"
+GDRIVE_EMAIL=slack@rpiambulance.com
```
Comment (ddbruce) [inline .env.example]: Please change this to an example email address

File: server.js [hunk 2]
```diff
@@ -30,16 +34,11 @@ app.action(
...
-    action: { action_id },
-    body: {
-      user: { id: user },
-      channel: { id: channel },
-      message: { ts, blocks }
-    }
-  }) => {
-    markChoreDone(action_id, user, channel, ts, blocks);
+  async ({ action, body }) => {
+    if (!action || !body || !body.user || !body.channel || !body.message) return;
```
Comment (ddbruce) [inline server.js]: Do we need to add anything for Sentry here?

File: server.js [hunk 3]
```diff
@@ -8,18 +9,21 @@ const { getTodaysChores } = require("./utilities/getChores.js");
 const { sendNoChores, postToSlack } = require("./utilities/notify.js");
 const { markChoreDone } = require("./utilities/markChoreDone.js");
 
-//package configuration
-
-//local configuration
-
-//globals
-const CRON_SCHEDULE = process.env.CRON_SCHEDULE;
```
Comment (ddbruce) [inline server.js]: Why did you remove this here rather than adding further globals? In other files, we make globals from `process.env` variables; here, we're not doing it. I prefer seeing all the globals in one place at the top of the file.

Comment (justetz) [reply server.js]: If you use environment variables directly instead of caching them in variables, it allows for the variables to be changed while the code is still running.

File: server.js (hunk 2 above)
Comment (justetz) [reply server.js]: In the future, we can add the following for all functions:

```js
Sentry.configureScope(scope => {
  scope.setUser({ username: "[username]" });
  scope.setExtra("key", "value");
  // etc...
});
```

File: server.js (hunk 3 above)
Comment (ddbruce) [reply server.js]: Thank you for mentioning this; I had not thought about it. You make a good point, except that that doing this would most often require the container to be `--force-recreate`d as the envs are stored in `docker-compose.yml`.

Comment (justetz) [reply server.js]: Fair point; however, I believe it is not necessary to store an environment variable in memory as a constant if it does not need a default value specified in the code.

Comment (ddbruce) [reply server.js]: I do realize doing it the way I mentioned is the storage of an extra var in memory.

Comment (ddbruce) [reply server.js]: ^ sounds good. I'll approve.
